docs/reference/api/storage/ChainingBlockDevice.md

```diff
@@ -1,6 +1,6 @@
 ## ChainingBlockDevice
 
-The ChainingBlockDevice class provides a way to chain together multiple block devices. You can then interact with the chained block devices as if they were a single block device of size equal to the sum of each substorage unit. 
+The ChainingBlockDevice class provides a way to chain together multiple block devices. You can then interact with the chained block devices as if they were a single block device of size equal to the sum of each substorage unit. The ChainingBlockDevice acts as an opposite of the [SlicingBlockDevice](/docs/v5.6/reference/slicingblockdevice.html).
```

Comment:
not sure the policy of linking and not a fan of linking but happy to be told otherwise. If our docs are well organized finding class names should be intuitive.

Reply:
Fair enough, I could pull the links out. I don't feel particularly strongly one way or the other.

Reply:
I don't mind including internal links that keep our users within our documentation. As a user, I find them helpful. Even if I can go to the navigation bar and find the API easily, it's nice not to have to. However, I do think too many links on a page is problematic.